extensions-core/histogram/src/main/java/io/druid/query/aggregation/histogram/sql/QuantileSqlAggregator.java

```diff
@@ -183,7 +183,7 @@ public Aggregation toDruidAggregation(
       }
     } else {
       final ExpressionVirtualColumn virtualColumn = input.toVirtualColumn(
-          String.format("%s:v", name),
+          StringUtils.format("%s:v", name),
```

Comment:
Forbidden-apis forbids uses of the default Locale, they were eliminated mostly in #4409, but some occurrences could have been missed, and some new were introduced since that PR was merged. This is mentioned in the PR description.